package.json

```diff
@@ -30,7 +30,7 @@
     "**/react": "16.5.2",
     "**/react-dom": "16.5.2",
     "**/browserslist": "^3.2.8",
-    "**/react-primitives/react-native-web": "0.9.3"
+    "**/react-primitives/react-native-web": "0.9.6"
```

Comment:
If react-primitives depends on the latest version of react-native-web, I’d rather remove this resolution altogether.

Reply:
Good call. Please see the updated PR.